Backend/Underdogs-Struggling/src/main/sql/DatabaseTables.sql

```diff
@@ -0,0 +1,45 @@
+
+-- Users: Store user information
+--     UserID (PK)
+--     UserName
+--     Password
+--     Role (FK, to Roles.RoleID)
+--     Email (Unique)
+--     RegistrationDate
+CREATE TABLE Users(UserID INT UNIQUE, Username VARCHAR(20) UNIQUE, Password VARCHAR(20) NOT NULL, Role INT , Email VARCHAR(255) UNIQUE, RegistrationDate Datetime, PRIMARY KEY (UserID));
+
+-- Roless: Store roles information
+--     RoleID (PK)
+--     RoleName
+CREATE TABLE Roles(RoleID INT, RoleName VARCHAR(20), PRIMARY KEY (RoleID));
+
+-- Entries: Store knowledge entries
+--     EntryID (PK)
+--     Title (Unique)
+--     Topic/SubTitle
+--     Order    (INT?) changed name to Index***
+--     Content
+--     AuthorID (FK, to Users.UserID)
+--     CreatedDate
+--     LastModifyDate
+--     KnowledgeLevel
+CREATE TABLE Entries(EntryID INT, Title VARCHAR(1111) UNIQUE, Topic VARCHAR(255), Index INT, Content VARCHAR(1111),
```

Comment:
For `Title`, I think 50 is enough LOL. But for `Content` we probably need more. We can discuss it on Sunday.